vraygolaem.h

```diff
@@ -54,11 +54,7 @@ enum {
 	pb_cache_name,
 	pb_cache_dir,
 	pb_character_files,
-	// motion blur
-	pb_motion_blur_enable,
```

Comment:
You can't do this. This will break loading of scenes saved with older versions of the plugin. Changes to this enum should happen only and only at its end. You can't insert new values in the middle of it, nor can you remove values that were present in a previous release.